src/main/java/com/github/nicholasmoser/gnt4/GNT4Workspace.java

```diff
@@ -129,7 +130,7 @@ public void revertFiles(Collection<String> filePaths) throws IOException {
       String fpkFilePath = file.fpkFilePath();
       if (fpkFilePath != null) {
         // FPK child file
-        Path saved = compressed.resolve(filePath);
+        Path saved = compressed.resolve(fpkFilePath);
```

Comment:
This is a bug fix for reverting files not related to costumes, it was introduced by https://github.com/NicholasMoser/GNTool/pull/124